### PR TITLE
[PSUPCLPL-12171] Make expect timeouts customized

### PR DESCRIPTION
**Pull request by n549**
### Description
Currently to change expect timeouts for pods and deployments in `plugins` it's necessary to copy the whole `procedures` part of the plugin, it is rather inconvenient.

We need a solution to allow customizing expect timeouts without repetition the `procedures` part of the plugin.

Fixes # (issue)
PSUPCLPL-12171

### Solution
Move `expect` settings for deployments, kubernetes and plugins pods from `globals.yaml` to `defaults.yaml`.
Now it's possible to change global expect timeouts for deployments, kubernetes and plugins pods.

This change affects only OOB plugins (calico, ingress-nginx, kubernetes-dashboard), because for custom plugins expect timeouts can be overridden in the `procedures` part the usual way.

### How to apply
-

### Test Cases
Deploy a cluster with cluster.yaml containing redefined values of expect timeouts, for example:
```
globals:
  expect:
    deployments:
      timeout: 3
      retries: 60
    pods:
      kubernetes:
        timeout: 4
        retries: 40
      plugins:
        timeout: 10
        retries: 80
```
ER: The cluster is deployed successfully, in the debug log there are lines like `Max expectation time:...` with proper values of `timeout*retries` seconds.

### Checklist
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] Integration CI passed
- [ ] Unit tests. If Yes list of new/changed tests with brief description
- [x] There is no merge conflicts

#### Unit tests
Indicate new or changed unit tests and what they do, if any.

### CODE REVIEW

File: documentation/Installation.md [hunk 1]
```diff
@@ -4311,6 +4311,24 @@ plugins:
                 - calico-node
 ```
 
+Default values for plugins pods expect timeout and retries are:
+
+|Configuration|Value|Description|
+|---|---|---|
+|timeout|`5`|The number of seconds until the next pod status check.|
+|retries|`150`|The number of attempts to check the status.|
```
Comment (ilia1243) [inline documentation/Installation.md:4319]: Is default value for retries 150 or 45?

File: kubemarine/resources/schemas/definitions/globals.json [hunk 2]
```diff
@@ -4,6 +4,72 @@
...
+            "timeout": {
+              "type": "integer",
+              "minimal": 1,
+              "default": 5,
+              "description": "Timeout for deployment expect check in seconds"
+            }
+          }
+        },
+        "pods" :{
+          "kubernetes": {
```
[the remark this answers is not in the document]
Comment (n549) [reply kubemarine/resources/schemas/definitions/globals.json]: Thanks, fixed.

File: documentation/Installation.md (hunk 1 above)
Comment (n549) [reply documentation/Installation.md:4319]: It's 150 (as it's needed for calico).

Comment (ilia1243) [reply documentation/Installation.md:4319]: Thank you. I will leave link to comment https://github.com/Netcracker/KubeMarine/pull/347#discussion_r1100063511 as it again becomes actual. If 150 retries are necessary only for calico, then we can leave this config in the calico configuration. Why it is again necessary to change default global timeout?

**Review by ilia1243 — APPROVED** [review]

**Pull request merged**